client/modules/accounts/templates/profile/profile.js

```diff
@@ -71,5 +71,9 @@ Template.accountProfile.helpers({
 
   isMarketplaceGuest: function () {
     return (Reaction.hasMarketplaceAccess("guest") && !Reaction.hasAdminAccess());
+  },
+
+  isMarketplaceSeller: function () {
+    return (Reaction.hasMarketplaceAccess() && !Reaction.hasOwnerAccess());
```

Comment:
There is always a case where Marketplace might be disabled as a module, or is enabled but doesn't allow for shops to be created if that one option is unchecked. This function covers these cases and makes sure the module is enabled, seller shops can be created, and the user has permission. Otherwise if any of these fail, access is false. We're not very fond of it, but we always had to check these two options along with permissions. What is needed is a global way to know whether the app is operating in marketplace mode or not. and then just have that checked in permissions for us every time. So hasPermission could do the job for us maybe?